client/etcd/client.go

```diff
@@ -109,23 +109,28 @@ func (c *csclient) KV() (kv.Store, error) {
 
 func (c *csclient) Txn() (kv.TxnStore, error) {
 	c.txnOnce.Do(func() {
-		c.txn, c.txnErr = c.createTxnStore(kvPrefix)
+		c.txn, c.txnErr = c.createTxnStore(
+			kv.NewOptions().
+				SetNamespace(kvPrefix).
+				SetZone(c.opts.Zone()).
```

Comment:
I'm not sure we want to use the zone from the options here as that is the zone that determines where the keys are stored but the zone here should indicate which zone the keys are for. Would prefer a complete separation of the two so they aren't confused.

Reply:
I'm ok with taking the zone option out now and figure out a solution later when we have concrete use cases.